CZI/viz_evaluation.md

```diff
@@ -1,34 +1,39 @@
 # ArviZ Visual Refresh, Python API Assessment, and other advice
-ArviZ is a Python and Julia libary for exploratory analysis of Bayesian models,
+ArviZ is a Python and Julia library for exploratory analysis of Bayesian models,
 and includes many [vizualizations](https://python.arviz.org/en/latest/examples/index.html)
 to that end.
 
 As with any open source package part of the challenge is maintaining an existing package.
 As part of the Chan Zuckerberg Initiative funds have been set aside for this express purpose.
 
 The scope of this work is to assess update visual style of the package (all components including docs)
-and perform an assessment of the API (primary python), and offer any other advice 
+and perform an assessment of the API (primary python), and offer any other advice
 
 ## Job description
-We are expecting a report covering the following points:
+We are expecting a report covering the following points (sorted in descending priority):
 
-* Analysis of available visualizations across Python and Julia
 
-  Evaluate and criticize current plots for visual style and effectiveness.
-  Critique the visual clarity of each
+1. Provide a design guide
 
-* Provide a design guide
-  This ideally would be in the form of an MPL, Bokeh, or other style template.
-  A README.md style guide should provided if deemed necessary as well.
+   This ideally would be in the form of a README.md style guide with explanation and references,
+   with focus around ArviZ usecases and bad practices we might be using now.
+   A matplotlib, Bokeh, or other style template could also be included as complementary material
 
-* Critique of the Python or Julia API 
-  This role will require you to get hands on with code. During this hands on process
-  document improvements that could be made to the API and code workflow itself
-  would be expected.
+1. Critique of the Python or Julia API
 
-* Any other advice or tips
-  In the course of performing this work, detailed issues tickets or reccomendations
-  for challenges or improvements would aid the ArviZ team in elevating the suite of projects
+   This role will require you to get hands on with code.
+   During this process documenting the improvements that could be made to the API and
+   code workflow itself is expected, updates to the code are not a priority.
+
+1. Analysis of available visualizations across Python and Julia
+
+   Evaluate and criticize current plots for visual style and effectiveness.
+   Critique the visual clarity of each
+
+1. Any other advice or tips
```

Comment:
* These are all marked one, are they the same priority or were they mean to be different